packages/libs/coreui/src/components/info/Tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import { ComponentProps } from 'react';
+import { Theme, Tooltip as MUITooltip, withStyles } from '@material-ui/core';
+import _ from 'lodash';
+
+/**
+ * Tooltip will not render if the title is an empty value or a boolean.
+ */
+const UnstyledTooltip = (props: ComponentProps<typeof MUITooltip>) => {
+  const { title } = props;
+
+  return !(
+    title === '' ||
+    _.isEqual(title, {}) ||
+    _.isEqual(title, []) ||
+    typeof title === 'boolean'
+  ) ? (
+    <MUITooltip {...props} />
+  ) : null;
+};
```

Comment:
So in [the checkbox tree of this UI](https://plasmodb.org/a/app/record/gene/PF3D7_0104400#SNPsAlignment), I don't think this logic is quite working yet. The title is being passed into the tooltip as `undefined`, which evaluates as true in the ternary, and results in an empty yellow box. When I tweak the `FieldNode` prop to render an empty string as a fallback to undefined, then evaluating to false in the ternary results in none of the content being rendered. BUT, I noticed in my testing that if the MUITooltip itself receives an empty string, it will render the desired content: the content itself with no tooltip. As a result, here's what I did that seems to achieve the desired result:

```js
const UnstyledTooltip = (props: ComponentProps<typeof MUITooltip>) => {
  const { title, ...remainingProps } = props;

  const finalProps = !(
    title === '' ||
    // I added this null-ish check to account for null or undefined
    title == null ||
    _.isEqual(title, {}) ||
    _.isEqual(title, []) ||
    typeof title === 'boolean'
  ) ? props : { title: '', ...remainingProps}
  
   return <MUITooltip {...finalProps} />
};
```